jqassistant.properties

```diff
@@ -1,21 +1,27 @@
 category=External Analyzers
 description=Analyze issues reported by jQAssistant.
 homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
-archivedVersions=1.7.2
-publicVersions=1.7.3
+archivedVersions=1.7.3
+publicVersions=1.8.1
  
 defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
 defaults.mavenArtifactId=sonar-jqassistant-plugin
 
+1.8.1.description=Upgraded to jQAssistant 1.8.0 
+1.8.1.date=2002-02-03
+1.8.1.sqVersions=[7.9,LATEST]
+1.8.1.changelogUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin/blob/master/release-notes.adoc#181
+1.8.1.downloadUrl=https://search.maven.org/remotecontent?filepath=org/jqassistant/contrib/sonarqube/sonar-jqassistant-plugin/1.8.1/sonar-jqassistant-plugin-1.8.1.jar
+
 1.7.3.description=Clarified behavior of configuration property `sonar.jqassistant.reportPath` and improved message for reported issues
 1.7.3.date=2019-12-19
-1.7.3.sqVersions=[7.9,LATEST]
+1.7.3.sqVersions=[7.9,8.1]
 1.7.3.changelogUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin/blob/master/release-notes.adoc#173
 1.7.3.downloadUrl=https://search.maven.org/remotecontent?filepath=org/jqassistant/contrib/sonarqube/sonar-jqassistant-plugin/1.7.3/sonar-jqassistant-plugin-1.7.3.jar
 
 1.7.2.description=Initial entry into the Marketplace
 1.7.2.date=2019-11-22
-1.7.2.sqVersions=[7.9,7.9*]
+1.7.2.sqVersions=[7.9,8.1]
```

Comment:
No need to change compatibility of old versions other than the immediately previous one b/c theoretically no one on 8.0 or 8.1 has this version installed. Please revert this line.

Reply:
Fixed.